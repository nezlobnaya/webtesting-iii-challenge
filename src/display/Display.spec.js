// Test away!
import React from 'react';
import Display from './Display';
import * as rtl from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer'

afterEach(rtl.cleanup)

test('should match the snapshot', () => {
    const tree = renderer.create(<Display />).toJSON()
    expect(tree).toMatchSnapshot()
})

test('a gate should default to `unlocked` and `open`', () => {
    const tree = renderer.create(<Display closed={false} locked={false}/>).toJSON()
    expect(tree).toMatchSnapshot()
})