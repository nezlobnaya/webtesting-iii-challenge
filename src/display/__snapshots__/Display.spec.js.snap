// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should default \`unlocked\` and \`open\` 1`] = `
<div
  className="display panel"
>
  <div
    className="led green-led"
  >
    Unlocked
  </div>
  <div
    className="led green-led"
  >
    Open
  </div>
</div>
`;

exports[`should default to \`unlocked\` and \`open\` 1`] = `
<div
  className="display panel"
>
  <div
    className="led green-led"
  >
    Unlocked
  </div>
  <div
    className="led green-led"
  >
    Open
  </div>
</div>
`;

exports[`should match the snapshot 1`] = `
<div
  className="display panel"
>
  <div
    className="led green-led"
  >
    Unlocked
  </div>
  <div
    className="led green-led"
  >
    Open
  </div>
</div>
`;
